### PR TITLE
fix: replace paths to assets globally

### LINKED ISSUE #36: [🐛] Bug: Path to asset gets updated only the first time
## Describe the bug
This is pretty straightforward: adding the same asset multiple times to an HTML file (for example) results in 404, because the path doesn't get updated globally.

```HTML
<img src="image.svg" />
<img src="image.svg" />
<img src="image.svg" />
```

becomes

```HTML
<img src="assets/image.c9a7e9b8.svg">
<img src="image.c9a7e9b8.svg">
<img src="image.c9a7e9b8.svg">
```

## Error messages

None.

## Expected behavior

The resulting index.html file should be updated globally:

```HTML
<img src="assets/image.c9a7e9b8.svg">
<img src="assets/image.c9a7e9b8.svg">
<img src="assets/image.c9a7e9b8.svg">
```
## Information
- Package version: v2.4.0
- Node version: v14.16.0

## Additional context
Please find a minimal test case here: https://github.com/NicolasReibnitz/simple-parcel-structure-test.

### PR DESCRIPTION
**Pull request by NicolasReibnitz**
Closes #36 

- Added polyfill for [String.prototype.replaceAll](https://github.com/es-shims/String.prototype.replaceAll)
- Switched `replace` with `replaceAll` to replace all occurrences of the moved file instead of only the first
- Ran the tests using `yarn test:watch`

### CODE REVIEW

**Comment by NicolasReibnitz** [comment]
Well, the added dependency and `--frozen-lockfile` don't seem to mix. Sorry.

File: src/core/FileManager.ts [hunk 1]
```diff
@@ -80,7 +81,7 @@ export class FileManager extends ConfigProvider {
         return dependents.reduce((newContent, dependent) => {
             const thisDependent = this.assetsMap.get(dependent);
             if (thisDependent?.replacer) {
-                newContent = newContent.replace(dependent, thisDependent.replacer);
+                newContent = replaceAll(newContent, dependent, thisDependent.replacer);
```
Comment (samrith-s) [inline src/core/FileManager.ts:84]: Actually, you do not need the `replaceAll` package or the function. You can simple change it back to what it was previously, with this slight modification.
```suggestion
                newContent = newContent.replace(new RegExp(dependent, 'g'), thisDependent.replacer);
```

And you can remove the dependency as well.

**Review by samrith-s — CHANGES_REQUESTED** [review]
The fix is very simple and doesn't actually need the dependency. You can check my comment.

File: src/core/FileManager.ts (hunk 1 above)
Comment (NicolasReibnitz) [reply src/core/FileManager.ts]: You are completely correct of course. Elegant solution. I applied your suggestions.

**Comment by samrith-s** [comment]
Thank you so much @NicolasReibnitz! I will release this ASAP 😄

**Comment by NicolasReibnitz** [comment]
That's great! Thank you! 😀

**Pull request merged**